app/src/main/java/io/gnosis/safe/utils/ImageUtils.kt

```diff
@@ -34,42 +32,10 @@ fun ImageView.loadTokenLogo(
             Picasso.get()
                 .load(icon)
                 .placeholder(placeholderDrawable)
-                .error(placeholderDrawable)
```

Comment:
what happens in case token logo can't be loaded?

Reply:
The asset set with placehoder() is shown.

Reply:
Do we get rid of the `error` asset? What happens if the URL in `icon` does not resolve into something Picasso can handle?